code/modules/spells/targeted/ethereal_jaunt.dm

```diff
@@ -44,7 +43,8 @@
 	target.incorporeal_move = 3
 	target.invisibility = INVISIBILITY_MAXIMUM
 	target.flags |= INVULNERABLE
-	target.alpha = 125 //Spoopy mode to know you are jaunting
+	target.alpha = 125
```

Comment:
can remove this line and put handle_alpha() just below where you add jaunting to alphas